embedded-selenium/src/main/java/com/playtika/test/selenium/EmbeddedSeleniumBootstrapConfiguration.java

```diff
@@ -104,9 +104,15 @@ public BrowserWebDriverContainer selenium(
             SeleniumProperties properties,
             MutableCapabilities capabilities
     ) {
-
-        BrowserWebDriverContainer container = new BrowserWebDriverContainer<>().withCapabilities(capabilities);
-        container.withRecordingFileFactory(getRecordingFileFactory());
+        BrowserWebDriverContainer container;
+        String imageName = properties.getImageName();
+        if(isNotBlank(imageName)) {
```

Comment:
better just
```
        BrowserWebDriverContainer container = isNotBlank(imageName)
? new BrowserWebDriverContainer<>(imageName)
:  new BrowserWebDriverContainer<>()
```
so you also not copypasting duplicate code

Reply:
ok 👍